From 44cec300b96fb4d3a23d84ce45fc6583493d2f7a Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Thu, 23 Nov 2017 13:51:07 -0600
Subject: [PATCH] "0001-security-don-t-build-keystore
 _______________________________

*Previous From fa17df7778bf9314c6181e37018ef59036fd6d73 Mon Sep 17 00:00:00 2001
*Previous From: Meticulus <theonejohnnyd@gmail.com>
*Previous Date: Fri, 17 Nov 2017 19:30:09 -0600
*Previous Subject: [PATCH] security: don't build keystore
*
*Change-Id: Idaed548934cf5df8815ca9f44accd481dc3842a3"

Change-Id: If57ed3d686ed2697fa6d54ef14118b0eb6443929
---
 keystore-engine/Android.mk | 1 +
 keystore/Android.mk        | 2 ++
 softkeymaster/Android.mk   | 3 ++-
 3 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/keystore-engine/Android.mk b/keystore-engine/Android.mk
index 1f5d903..ff12d74 100644
--- a/keystore-engine/Android.mk
+++ b/keystore-engine/Android.mk
@@ -57,4 +57,5 @@ LOCAL_SHARED_LIBRARIES += \
 
 LOCAL_ADDITIONAL_DEPENDENCIES := $(LOCAL_PATH)/Android.mk
 
+LOCAL_MODULE_RELATIVE_PATH := notused
 include $(BUILD_SHARED_LIBRARY)
diff --git a/keystore/Android.mk b/keystore/Android.mk
index baff509..58e0b4b 100644
--- a/keystore/Android.mk
+++ b/keystore/Android.mk
@@ -59,6 +59,7 @@ LOCAL_C_INCLUES := system/keymaster/
 LOCAL_CLANG := true
 LOCAL_SANITIZE := integer
 LOCAL_ADDITIONAL_DEPENDENCIES := $(LOCAL_PATH)/Android.mk
+LOCAL_MODULE_RELATIVE_PATH := notused
 include $(BUILD_EXECUTABLE)
 
 include $(CLEAR_VARS)
@@ -116,6 +117,7 @@ LOCAL_EXPORT_C_INCLUDE_DIRS := $(LOCAL_PATH)/include
 LOCAL_CLANG := true
 LOCAL_SANITIZE := integer
 LOCAL_ADDITIONAL_DEPENDENCIES := $(LOCAL_PATH)/Android.mk
+LOCAL_MODULE_RELATIVE_PATH := notused
 include $(BUILD_SHARED_LIBRARY)
 
 # Library for unit tests
diff --git a/softkeymaster/Android.mk b/softkeymaster/Android.mk
index eb32c87..fa1b415 100644
--- a/softkeymaster/Android.mk
+++ b/softkeymaster/Android.mk
@@ -19,7 +19,7 @@ ifeq ($(USE_32_BIT_KEYSTORE), true)
 LOCAL_MULTILIB := 32
 endif
 LOCAL_MODULE := keystore.default
-LOCAL_MODULE_RELATIVE_PATH := hw
+LOCAL_MODULE_RELATIVE_PATH := notused
 LOCAL_SRC_FILES := module.cpp
 LOCAL_C_INCLUDES := system/security/keystore
 LOCAL_CFLAGS = -fvisibility=hidden -Wall -Werror
@@ -38,6 +38,7 @@ LOCAL_C_INCLUDES := system/security/keystore \
 	$(LOCAL_PATH)/include
 LOCAL_CFLAGS = -fvisibility=hidden -Wall -Werror
 LOCAL_SHARED_LIBRARIES := libcrypto liblog libkeystore_binder
+LOCAL_MODULE_RELATIVE_PATH := notused
 LOCAL_MODULE_TAGS := optional
 LOCAL_EXPORT_C_INCLUDE_DIRS := $(LOCAL_PATH)/include
 LOCAL_ADDITIONAL_DEPENDENCIES := $(LOCAL_PATH)/Android.mk
-- 
2.7.4

